test/functional/normal/K_spec.lua

```diff
@@ -29,7 +27,7 @@ describe('K', function()
   it("invokes non-prefixed 'keywordprg' as shell command", function()
     helpers.source([[
       let @a='fnord'
-      set keywordprg=echo\ fnord\ >>]])
+      set keywordprg=echo\ fnord]] .. (helpers.iswin() and '>>' or '\\ >>'))
```

Comment:
If the extra space was causing a problem on windows, it's ok (and probably less confusing) to remove it for all platforms.